core/domain/exp_jobs_one_off.py

```diff
@@ -608,3 +616,53 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class ImageDataMigrationJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for migrating the images in the exploration
+    from the GAE to GCS.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [file_models.FileSnapshotContentModel]
+
+    @staticmethod
+    def map(file_snapshot_content_model):
+        instance_id = (
+            file_snapshot_content_model.get_unversioned_instance_id())
+        filetype = instance_id[instance_id.rfind('.') + 1:]
+        # To separate the image entries from the audio entries we get from the
+        # FileSnapshotContentModel.
+        if filetype in ALLOWED_IMAGE_EXTENSIONS:
+            pattern = re.compile(r'^/([^/]+)/assets/(([^/]+)\.(' + '|'.join(
+                ALLOWED_IMAGE_EXTENSIONS) + '))$')
+            catched_groups = pattern.match(instance_id)
+            if not catched_groups:
+                yield('Error: The instance_id is not correct', instance_id)
```

Comment:
Make constant for this error string too.

Reply:
Done!